magda-web-server/src/index.ts

```diff
@@ -213,7 +213,7 @@ const topLevelRoutes = [
     "sign-in-redirect",
     "dataset",
     "projects",
-    "publishers"
+    "organisations"
```

Comment:
Good catch!

However it'd probably be best to leave both of these so that react-router can handle the routing.